infra/nightly-resources/index.js

```diff
@@ -1,346 +1,313 @@
-
-
 // copied from profile.py
-const allmodes = [["rvsdg_roundtrip", "rvsdg-round-trip-to-executable", ""],
-
-["egglog_noopt_brilift_noopt", "compile-brilift", "--optimize-egglog false --optimize-brilift false"],
-["egglog_noopt_brilift_opt", "compile-brilift", "--optimize-egglog false --optimize-brilift true"],
-["egglog_opt_brilift_noopt", "compile-brilift", "--optimize-egglog true --optimize-brilift false"],
-["egglog_opt_brilift_opt", "compile-brilift", "--optimize-egglog true --optimize-brilift true"],
-
-["egglog_noopt_bril_llvm_noopt", "compile-bril-llvm", "--optimize-egglog false --optimize-bril-llvm false"],
-["egglog_noopt_bril_llvm_opt", "compile-bril-llvm", "--optimize-egglog false --optimize-bril-llvm true"],
-["egglog_opt_bril_llvm_noopt", "compile-bril-llvm", "--optimize-egglog true --optimize-bril-llvm false"],
-["egglog_opt_bril_llvm_opt", "compile-bril-llvm", "--optimize-egglog true --optimize-bril-llvm true"]]
-
-
-var enabledModes = new Set();
-// a list of warnings
-var warnings = [];
+const allModes = [
+  "rvsdg_roundtrip",
+  "egglog_noopt_brilift_noopt",
+  "egglog_noopt_brilift_opt",
+  "egglog_opt_brilift_noopt",
+  "egglog_opt_brilift_opt",
+  "egglog_noopt_bril_llvm_noopt",
+  "egglog_noopt_bril_llvm_opt",
+  "egglog_opt_bril_llvm_noopt",
+  "egglog_opt_bril_llvm_opt",
+];
+
+const GLOBAL_DATA = {
+  enabledModes: new Set(),
+  warnings: new Set(),
+  currentRun: {},
+  baselineRun: undefined,
+};
 
 function addWarning(warning) {
```

Comment:
should addWarning be in handlers? not sure the precise destinction

Reply:
Nope- handlers is for stuff that gets called from a browser event (like on page load, on dropdown change, checkbox, etc)